kong/router.lua

```diff
@@ -388,15 +388,7 @@ do
           end
 
           -- plain or prefix match from the index
-          if route_t.strip_uri then
```

Comment:
This branch was an optimization around the two `sub` and the concatenation operations that it enclosed. Depending on the answer to the comment right above this one (about `ctx.matches.stripped_uri`), we may or may not need bring it back.